### ISSUE TITLE
MODULE_NOT_FOUND error if used with namespaced @hapi/hapi package

### ISSUE THREAD

**Issue by illarionvk** [simov/grant#117]
The [Hapi version check][version-check] in Grant causes MODULE_NOT_FOUND error if Hapi package is installed as `@hapi/hapi`.

```
Error: Cannot find module 'hapi/package.json'
    at Function.Module._resolveFilename (internal/modules/cjs/loader.js:582:15)
    at Function.Module._load (internal/modules/cjs/loader.js:508:25)
    at Module.require (internal/modules/cjs/loader.js:637:17)
    at require (internal/modules/cjs/helpers.js:22:18)
    at Object.exports.hapi (/app/node_modules/.registry.npmjs.org/grant/4.5.1/node_modules/grant/grant.js:12:26)
    at Object.registerPlugins (/app/lib/server/plugins.js:51:37)
    at Object.initServer (/app/lib/server/index.js:32:21)
    at provision (/app/lib/index.js:8:35)
    at Object.<anonymous> (/app/lib/index.js:22:1)
    at Module._compile (internal/modules/cjs/loader.js:701:30)
```

I'm working around this issue by requiring the consumer file directly:

```js
var grant = require('grant/build/lib/consumer/hapi17')
```

Hapi universe started moving npm packages to `@hapi/` namespace since Hapi v18.2.

[version-check]: https://github.com/simov/grant/blob/d44968eeb232f2c7d9a7803f9997e44e020ab3ee/grant.js#L12

**Comment by simov** [comment]
Thanks for the feedback, @illarionvk! Yes, I know they are moving into their own namespace. I'll push a fix shortly.

**Comment by simov** [comment]
Fixed in v4.5.2
